### ISSUE TITLE
Wrong list adjusted in IPF process

### ISSUE THREAD

**Issue by asongtoruin** [Transport-for-the-North/caf.base#73]
In the IP process, `targets_out` is created to provide the output targets. However, `reference` is removed from the `targets` list, which is then not used:

https://github.com/Transport-for-the-North/caf.base/blob/f5704cffd11ed7a4e9e6a68683bc994adf110656/src/caf/base/data_structures.py#L2370-L2375

I'm fairly sure line 2374 should refer to `targets_out`. @isaac-tfn can you confirm?

**Comment by isaac-tfn** [comment]
Yes you're right I'll change that now - this hopefully hasn't been breaking anything in land use it just may have been causing excess run times as a target would have been adjusted to twice per iteration potentially
